README.rst

```diff
@@ -105,6 +105,21 @@ This new template will then be used to render the /donate url.
 Troubleshooting
 ---------------
 
+Can't override styles using Indigo Theme for MFEs
+-------------------------------------------------
+
+The indigo theme can’t override styles for MFEs directly. It overrides the styles for edx-platform. In case of MFEs, `@edx/brand <https://github.com/openedx/brand-openedx>`_ is used to override the styles. Then, customize the ``@edx/brand`` package to your preferences and include this customized package in `tutor-indigo` plugin. In this way, styles can be overidden::
```

Comment:
"Then, customize the" --> "Customize the". 'Then' is not making a sense at the start of the sentence.

Reply:
Done